analyzer.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+# for details. All rights reserved. Use of this source code is governed by a
+# BSD-style license that can be found in the LICENSE file.
+
+runtime: custom
+env: flex
+service: analyzer
+env_variables:
```

Comment:
Instead of relying on such a special variable, it might be better to use the `GAE_SERVICE` environment variable.